cg/src/domains/management/subjects/subjects.controller.js

```diff
@@ -17,14 +17,18 @@ class SubjectsController {
   }
 
   async listSubjects(req, res) {
-    const subjectsPage = await this.subjectsService.listSubjects(req.query.page);
-    subjectsPage.map(subject => {
+    const subjectsInfo = await this.subjectsService.listSubjects(req.query.page);
```

Comment:
Are these changes part of your PR or do you need to properly rebase @fjcero?

Reply:
nope, not sure what happened with the rebase